portfolio/src/main/webapp/photography.html

```diff
@@ -44,7 +64,9 @@ <h2>Monte Carlo, Monaco</h2>
         <a href="images/mc6.jpg"><img src="images/mc6.jpg"/></a>
         <a href="images/mc7.jpg"><img src="images/mc7.jpg"/></a>
       </div><div class="image-thumbnails">
-                <h2>Èze, France</h2>
+           <div text-align:left>
```

Comment:
here and on lines 80 and55,  text-align:left would need to be in a style property.

But it's better to wrap it in a class. if all divs within .image-thumbnails should have left-aligned text, you can use `.image-thumbnails div { text-align: left; }`.

Reply:
Done